lib/vsc/ldap/utils.py

```diff
@@ -491,6 +491,9 @@ def __modify(self, current, dn, attributes):
         current_ = {}
         for key in attributes.keys():
             current_[key] = current.get(key, [])
+            if current_[key] is '':
+                self.log.warning("Replacing empty string for key %s with EMPTYGECOSDURINGMODIFY before making modlist for dn %s" % (key, dn))
+                current_[key] = "EMPTYGECOSDURINGMODIFY"  # hack to allow replacing empty strings
```

Comment:
It is not, it's only needed for the underlying LDAP lib to not kick out the proposed change/addition.